composer.json

```diff
@@ -12,14 +12,13 @@
     "require": {
         "php": ">=7.1.0",
         "silverstripe/recipe-plugin": "^1",
-        "silverstripe/recipe-cms": "^4.4.1",
-        "silverstripe/admin": "1.4.2 as 1.4.1",
+        "silverstripe/recipe-cms": "^4.6",
         "silverstripe/bambusa-theme": "dev-master",
         "silverstripe/recipe-content-blocks": "^2.3",
         "silverstripe/crontask": "^2.1.2",
         "silverstripe/dynamodb": "^4.0.2",
         "dnadesign/silverstripe-elemental-userforms": "^3.0.0",
-        "jonom/silverstripe-betternavigator": "dev-pulls/auto-injection",
+        "jonom/silverstripe-betternavigator": "~5.0.1",
         "silverstripe/sharedraftcontent": "^2.3",
         "jonom/focuspoint": "~3.1.1",
```

Comment:
Might as well just change this to `^3`

Reply:
That was a conscious decision. This project is meant to self-upgrade ... we decided to use the looser `^` constraint on silverstripe dependencies because we're reasonably confident we know what we are doing (at least most of the time) and won't introduce breaking changes in a minor release.

However, with third party module we don't know that for sure. So we decided to use `~` constraint instead there because we wanted the decision to upgrade to a new minor to be conscious.